packages/deploy-script-support/src/coreProposalBehavior.js

```diff
@@ -54,37 +54,55 @@ export const makeCoreProposalBehavior = ({
     return fromEntries(ents);
   };
 
-  /** @param {ChainBootstrapSpace & BootstrapPowers & { evaluateInstallation: any }} allPowers */
+  /** @param {ChainBootstrapSpace & BootstrapPowers & { evaluateBundleCap: any }} allPowers */
   const behavior = async allPowers => {
     const {
-      consume: { board, agoricNamesAdmin },
-      evaluateInstallation,
+      consume: { vatAdminSvc, zoe, agoricNamesAdmin },
+      evaluateBundleCap,
       installation: { produce: produceInstallations },
       modules: {
         utils: { runModuleBehaviors },
       },
     } = allPowers;
     const [exportedGetManifest, ...manifestArgs] = getManifestCall;
 
-    const restoreRef = overrideRestoreRef || (x => E(board).getValue(x));
+    const defaultRestoreRef = async args => {
+      let p;
+      if (args.bundleName) {
+        p = E(vatAdminSvc).getNamedBundleID(args.bundleName);
+      } else {
+        p = Promise.resolve(args.bundleID);
+      }
```

Comment:
`tsc` drops `p` into the `any` bucket when it sees `let p`. Using a `?` expression and `const` rather than an `if` statement would address that.

Not critical.